bundles/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/PLCLogoBinding.java

```diff
@@ -288,8 +288,10 @@ protected void internalReceiveCommand(String itemName, Command command) {
 
             PLCLogoMemoryConfig wr = config.getWR();
             int address = -1;
+            int bit = -1;
             try {
                 address = wr.getAddress(controller.getModel());
+                bit = wr.getBit(controller.getModel());
```

Comment:
I would place another try catch block for bit calculation after this one.
Reason;  Get more clue what goes wrong in error - case